Dockerfile

```diff
@@ -1,35 +1,36 @@
 # setup OS and timezone
-FROM ubuntu:18.04
+FROM php:7.2-apache
```

Comment:
Why version 7.2?

Reply:
It was the php version installed previously with `apt-get install -y php7.2`